src/app.ts

```diff
@@ -40,6 +41,7 @@ app.use(`${pageURLs.STRIKE_OFF_OBJECTIONS}/*`, authenticationMiddleware);
 app.use(`${pageURLs.STRIKE_OFF_OBJECTIONS}/*`, objectionSessionMiddleware);
 // apply our default router to /
 app.use(pageURLs.STRIKE_OFF_OBJECTIONS, router);
+app.use(...errorHandler);
```

Comment:
at the moment, the error handler is default exporting an array of one function
export default [errorHandler];
so we should remove the square brackets and then we can remove the three dots here which expands the array (we won't be exporting an array if we remove the square brackets)
In extensions we had an array of 2 functions exported from the controller, the extra one for a 404 page. I can raise the 404 as a separate story.